Create WinGet package for June 2024 GDK Update 1

aaron-stebner opened the pull request:
Create WinGet package for June 2024 GDK Update 1

Tested locally with winget validate and winget install
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/165455)

aaron-stebner committed:
Create WinGet package for June 2024 GDK Update 1

RuthUlloa commented:
LGTM.

stephengillie commented:
URL: https://github.com/microsoft/GDK/archive/refs/tags/June_2024_Update_1.zip 
Status Code: 200

(Automated message - build 887)

aaron-stebner commented:
The license file was inadvertantly removed via https://github.com/microsoft/GDK/commit/c8db4b885f050a057f84835752ecf95a422c7eb3 but we've added it back via https://github.com/microsoft/GDK/commit/33695e711e346165e4eb43868ef1cd2803a2274f and the link at https://github.com/microsoft/GDK/blob/2406_JUNE/LICENSE-EN-US.MD works correctly now.

> Url validation error
> 
> * manifests\m\Microsoft\GDK\2406\240601
>   
>   * https://github.com/microsoft/GDK/blob/2406_JUNE/LICENSE-EN-US.MD
>     
>     * Http status code: NotFound

The license file was inadvertantly removed via https://github.com/microsoft/GDK/commit/c8db4b885f050a057f84835752ecf95a422c7eb3 but we've added it back via https://github.com/microsoft/GDK/commit/33695e711e346165e4eb43868ef1cd2803a2274f and the link at https://github.com/microsoft/GDK/blob/2406_JUNE/LICENSE-EN-US.MD works correctly now.
